projects/js-packages/components/components/boost-score-graph/uplot-line-chart.tsx

```diff
@@ -95,8 +95,8 @@ export default function UplotLineChart( { data, range, periods }: UplotChartProp
 	const uplot = useRef< uPlot | null >( null );
 	const uplotContainer = useRef( null );
 
-	const lastDesktopScore = data[ 1 ][ data[ 1 ].length - 1 ];
-	const lastMobileScore = data[ 2 ][ data[ 2 ].length - 1 ];
+	const lastDesktopScore = typeof data[ 1 ] !== 'undefined' ? data[ 1 ][ data[ 1 ].length - 1 ] : 0;
+	const lastMobileScore = typeof data[ 2 ] !== 'undefined' ? data[ 2 ][ data[ 2 ].length - 1 ] : 0;
```

Comment:
Might be worth describing exactly what `uPlot.AlignedData` is.

This `typeof` check doesn't inspire confidence. If it can be undefined, then may be it can be other types too?